projects/plugins/jetpack/extensions/blocks/button/view.scss

```diff
@@ -1,3 +1,7 @@
 .amp-wp-article .wp-block-jetpack-button {
 	color: #ffffff;
 }
+
+.wp-block-jetpack-button button {
+	border: inherit;
```

Comment:
This change seems to be related to https://github.com/Automattic/themes/issues/5776#issuecomment-1080932897